rescript-syntax/ppx_rescript_compat/ppx_rescript_compat.ml

```diff
@@ -47,10 +47,11 @@ let expr_mapper (self : mapper) (expr : Parsetree.expression) =
             args;
       }
   | Pexp_send
-      ( ({ pexp_desc = Pexp_apply _ | Pexp_ident _; pexp_loc; _ } as subexpr),
+      ( ({ pexp_desc = Pexp_apply _ | Pexp_ident _ | Pexp_send _; pexp_loc; _ } as subexpr),
```

Comment:
sure this fixes the case `((entry ## person) ## age)`, but does it fix for more `##`?

Reply:
it does, added another level of depth in https://github.com/melange-re/melange/pull/533/commits/2ca2df93982008272d58b47b4e2c9aa3fb6890cd.